src/config/dynamic-theme-fixes.config

```diff
@@ -585,6 +585,27 @@ body[style] {
 
 ================================
 
+bestbuy.ca
+
+INVERT
+.centerContainer
+
+CSS
+[class^="facetName"], [class^="title"], [class^="sortLabel"], [class^="headerText"], [class^="subTitle"] {
+    filter:invert(100%) hue-rotate(180deg) contrast(90%) !important;
+}
+.backgroundContainer_ryHAa {
+    z-index:0 !important;
+}
+.featureBanner_8BLDb .centerContainer {
+    z-index: 1 !important;
+}
+body {
```

Comment:
Can't find 123. A page where this is needed.

Reply:
https://www.bestbuy.ca/en-ca/category/gaming-monitors/474835
In order to make the banner appear (in this case, with the two monitors), the z-index of this element must be increased, along with the **.backgroundContainer_ryHAa element**.